src/SlackInterface.js

```diff
@@ -4,27 +4,133 @@
 
 /* eslint no-console:0 */
 
-var RtmClient = require('@slack/client').RtmClient;
-var RTM_EVENTS = require('@slack/client').RTM_EVENTS;
+/*******************************************************************************
+ * Globals
+ */
+const RtmClient = require('@slack/client').RtmClient;
+const RTM_EVENTS = require('@slack/client').RTM_EVENTS;
+const AWS = require('./AWS_API.js');
+var rtm;
+
+const token = process.env.SLACK_API_TOKEN || '';
+const DEBUG = process.env.DEBUG || false;
+AWS.DEBUG = DEBUG;
+
+
+/*******************************************************************************
+ * Functions
+ */
 
+var handleRtmMessage = function(message) {
+    if (DEBUG) { console.log('Message:', message) }
+
+    var text = message.text;
+
+    if (keyMessage(text, 'aws ')) {
+        text = text.substring('aws '.length, text.length);
+        if (keyMessage(text, 'describe ec2 ')) {
+            text = text.substring('describe ec2 '.length, text.length);
+            if (keyMessage(text, 'instance ')) {
+                AWS.checkEC2Instance(text.substring('instance '.length, text.length)).then(function (resp) {
+                    rtm.sendMessage(resp, message.channel);
+                }, function (err) {
+                    rtm.sendMessage('Looks like there was a problem processing your request', message.channel);
```

Comment:
That would be a good idea. Basically do
var problemMessage = 'Looks like there was a problem processing your request.';

and then just put rtm.sendMessage(problemMessage...

I'll make that change tonight